<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DividirCuenta</class>
 <widget class="QWidget" name="DividirCuenta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tablaCuenta">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>660</width>
     <height>520</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border:none;
selection-background-color: #46B04A;
}</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1021</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Encabezados2">
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="elPatio">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 133, 133);
</string>
      </property>
      <property name="text">
       <string>El Patio</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Encabezados">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="nombreMesero">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>#Mesero</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>651</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="MenuTop">
    <item>
     <layout class="QVBoxLayout" name="LetrasMenu">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="elPatio_2">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #585858
</string>
        </property>
        <property name="text">
         <string>Cuenta a Pagar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="elPatio_3">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>Selecciona si la Cuenta Ser√° Dividida</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="BtnsMenu">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnDividir">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}</string>
        </property>
        <property name="text">
         <string>Dividir Cuenta</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPagar">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
	color: #539E56;
	border-radius: 10px;
	background-color: #E3EFB0;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}</string>
        </property>
        <property name="text">
         <string>Pagar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
